modules/reference/pages/systems/system-details/sd-hardware.adoc

```diff
@@ -4,4 +4,6 @@
 The menu:Systems[Details > Hardware] section shows detailed hardware information about the selected client, including architecture, networking, and memory.
 
 Click btn:[Schedule Hardware Refresh] to update this page with the latest hardware information.
-To change the primary network interface on the client, select the correct interface in the [guimenu]``Primary Network Interface`` field in the [guimenu]``Networking`` section, and click btn:[Update Properties].
+To change the primary network interface on the client, select the correct interface in the [guimenu]``Primary Network Interface`` field in the [guimenu]``Networking`` section.
+For clients with multiple fully qualified domain names (FQDNs) configured, change the primary FQDN by selecting it in the [guimenu]``FQDNs`` field in the [guimenu]``Networking`` section.
```

Comment:
@Loquacity The GUI label/field was renamed from `FQDN` to `Primary FQDN` in that PR:
![107793387-24f75e00-6d4e-11eb-9214-378c32f62231](https://user-images.githubusercontent.com/14297426/108821603-c46de980-75b5-11eb-965e-9465ca331d97.png)